src/cascadia/TerminalApp/AppKeyBindings.idl

```diff
@@ -1,16 +1,9 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
+import "../ActionArgs.idl";

```

Comment:
is this bit necessary, or can midl figure it out?

Reply:
> can midl figure it out 
![image](https://user-images.githubusercontent.com/18356694/63172145-a4458d80-c002-11e9-9bea-5de311f7bc42.png)